utils/anautilities.py

```diff
@@ -384,16 +385,16 @@ def getGemType(entry):
     print("Writing output data")
 
     # Write out the dacVal results to a root file, a text file, and the terminal
-    outputTxtFiles_dacVals = ndict()
+    outputTxtFilenames_dacVals = ndict()
     for idx in range(len(dacNameArray)):
         dacName = np.asscalar(dacNameArray[idx])
         for entry in crateMap:
             ohKey = (entry['shelf'],entry['slot'],entry['link'])
             detName = getDetName(entry)
             if scandate == 'noscandate':
-                outputTxtFiles_dacVals[dacName][ohKey] = open("{0}/{1}/NominalValues-{2}.txt".format(elogPath,detName,dacName),'w')
+                outputTxtFilenames_dacVals[dacName][ohKey] = "{0}/{1}/NominalValues-{2}.txt".format(elogPath,detName,dacName)
             else:
-                outputTxtFiles_dacVals[dacName][ohKey] = open("{0}/{1}/dacScans/{2}/NominalValues-{3}.txt".format(dataPath,detName,scandate,dacName),'w')
+                outputTxtFilenames_dacVals[dacName][ohKey] = "{0}/{1}/dacScans/{2}/NominalValues-{3}.txt".format(dataPath,detName,scandate,dacName)
```

Comment:
Can't this block of code be advantageously merged with the for-loop line 416; I believe it would improve readability.

Reply:
Changed